enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -88,6 +90,61 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
         </div>
       </div>
     </div>
-
+    <div id="course-details-modal" class="modal" aria-hidden="true">
+      <div id="course-details-modal-content" class="modal-content" role="dialog" aria-modal="true" aria-labelledby="modal-header-text">
+        <button class="modal-close-button" id="modal-close-button" aria-label="${_('Close')}">${_('Close')}</button>
+        <header class="wrapper">
+          <div class="image">
+            <img src="${ course_image_uri }" alt="${ course_name }"/>
+          </div>
+          <div class="details">
+            <h1 id="modal-header-text">${ course_name }</h1>
+            <p class="short-description">${ course_short_description }</p>
+            <div class="enterprise">
+              <img src="${static.url('images/logo.png')}" alt="${ course_organization }" />
+            </div>
+          </div>
+        </header>
+        <div class="body">
+          <div class="details">
+            <ol>
+              <li>
+                <span class="icon fa fa-tag" aria-hidden="true"></span>
+                <span class="title">${ price_text}</span>
+                <span class="text">
+                  % if course_modes:
+                    % if course_modes[0]['final_price'] and course_modes[0]['original_price'] != course_modes[0]['final_price']:
+                    <strike>${ course_modes[0]['original_price'] }</strike>
+                    <br />
+                    <span class="discount">${ course_modes[0]['final_price'] }</span>
+                    % else:
+                    ${ course_modes[0]['original_price'] }
+                    % endif
+                  % endif
+                </span>
+              </li>
+              <li>
+                <span class="icon fa fa-certificate" aria-hidden="true"></span>
+                <span class="title">${ level_text}</span>
+                <span class="text">Intermediate</span>
```

Comment:
So, this actually isn't a detail that's in the existing data we're using elsewhere, so I can't easily patch it in from my PR; you'll need to make a call to the Course Discovery API and use the value of the `level_type` key for the course run.